lib/GenomeFileUtil/GenomeFileUtilImpl.py

```diff
@@ -473,6 +478,78 @@ def export_genome_features_protein_to_fasta(self, ctx, params):
         # return the results
         return [output]
 
+    def export_metagenome_as_gff(self, ctx, params):
+        """
+        :param params: instance of type "ExportParams" (input and output
+           structure functions for standard downloaders) -> structure:
+           parameter "input_ref" of String
+        :returns: instance of type "ExportOutput" -> structure: parameter
+           "shock_id" of String
+        """
+        # ctx is the context object
+        # return variables are: output
+        #BEGIN export_metagenome_as_gff
+        if 'input_ref' not in params:
+            raise ValueError('Cannot run export_metagenome_as_gff- no "input_ref" '
+                             'field defined.')
+
+        # get WS metadata to get ws_name and obj_name
+        ws = Workspace(url=self.cfg.workspaceURL)
+        ws_obj = ws.get_objects2({'objects': [{
+            'ref': params['input_ref'],
+            'included':['/assembly_ref', '/gff_handle_ref']}
+        ]})['data'][0]
+
+        info = ws_obj['data']
+        ws_info = ws_obj['info']
+
+        # export to file (building from KBase Genome Object)
+        result = self.metagenome_to_gff(ctx, {
+            'metagenome_ref': params['input_ref']
+        })[0]
+
+        # get assembly
+        if 'assembly_ref' in info:
+            assembly_ref = info['assembly_ref']
+        else:
+            raise ValueError("no 'assembly' associated with this AnnotatedMetagenomeAssembly "
```

Comment:
Can you change to "No Assembly associated with this AnnotatedMetagenomeAssembly...'